process/charter.html

```diff
@@ -178,13 +178,16 @@ <h2>3. <a id="creation" name="creation">Charter Creation</a></h2>
 
 <h3 id="horizontal-review">3.1 Horizontal Review</h3>
 
-	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:
-	<ol><li>Opening a <a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">Chartering issue</a> in the <a href="https://github.com/w3c/strategy/issues">Strategy Repo</a>,</li>
+	<p>Once there is agreement on a draft charter, ideally among Team Contact(s), candidate chairs(s), relevant interest groups, and interested community members, seek review from the Strategy Team and horizontal reviewers by:</p>
+	<ol>
+  <li>Moving/Copying the draft charter into the <a href="https://github.com/w3c/charter-drafts/">charter-drafts Repo</a>, naming it using the convention [YYYY]/[shortname-name]-[group-type].html .</li>
+  <li>Opening a <a href="https://github.com/w3c/strategy/issues/new?assignees=&labels=Evaluation%3A+untriaged&template=04-Chartering.md&title=">Chartering issue</a> in the <a href="https://github.com/w3c/strategy/issues">Strategy Repo</a>,</li>
 	<li>Adding a card for the issue to the <a href="https://github.com/w3c/strategy/projects/2">Incubation Pipeline</a>,</li>
 	<li>Adding the <a class="IssueLabel-big" style="background: #9ae504;" href="https://github.com/w3c/strategy/issues">Horizontal review requested</a>
 	label to the issue, and</li>
-	<li>Optionally, announce the new charter-in-progress to team-strategy and/or on a Strategy Team call.</li>
-	</ol></p>
+  <li>Creating an issue in <a href="https://github.com/w3c/AB-memberonly">w3c/AB-memberonly</a> to alert the AB and our Members.</li>
```

Comment:
Currently charter announcements go to the AC and chairs mailing lists, why have a different mechanism for this?

Reply:
I wonder whether copies of charters in different places will create confusion. I wouldn't mind the following:

1) Before their adoption, charters live on GitHub.
2) Once adopted, there's a snapshot copied to w3.org.
3) The GitHub instance is marked as superseded with a link to the version on w3.org.

Reply:
@chrisn , the optional announcement is part of the internal team process. We don't expect everyone in the Team to follow AC or chairs mailing list.

Reply:
@ianbjacobs the proposed (during AC review) and approved charter do live on w3.org, including for historical . This proposal only intends to clarify where the GitHub instance lives while it's in development.

Reply:
Hi @plehegar,

The proposal says "Moving/Copying the draft charter..." That's the part that I want to avoid. Rather than moving or copying, all draft charters should just be on GitHub.

Reply:
I'm not worried about internal team things (my comment on that was only to point out whether `team-strategy` needs a link) - rather the use of AB-memberonly for alerting the AB and members

Reply:
I'm not sure why we should be having issues in AB-memberonly to track reviews - I think these SHOULD be filed, but it would seem natural to put them in the charter-drafts repo, with clear indications when stages change?